hr-HR.lproj/Menu.strings

```diff
@@ -41,7 +41,7 @@
 "Browse All Versions" = "Pregledaj sve verzije";
 "Import" = "Uvezi";
 "Export" = "Izvezi";
-"Page Setup" = "Postavke Stranice";
+"Page Setup" = "Postavke stranice";
```

Comment:
Is menu label also like "Postavke stranice" instead of "Postavke Stranice" on built-in applications in your system?

Reply:
Yes.
HR in general has mostly _Sentence case_ labels.